pkg/wgembed/iface_linux.go

```diff
@@ -36,7 +37,11 @@ func (wg *commonInterface) Up() error {
 		return errors.Wrap(err, "failed to bring wireguard interface up")
 	}
 
-	if err := netlink.LinkSetMTU(link, 1420); err != nil {
+	MTU := device.DefaultMTU
```

Comment:
There is also some `device.DefaultMTU` being set in these other files:
- [‎pkg/wgembed/iface_kernel_linux.go](https://github.com/freifunkMUC/wg-embed/blob/20165f846d32c03c5517b628890e8eea4bdeca28/pkg/wgembed/iface_kernel_linux.go#L24)
- [pkg/wgembed/iface_userspace.go](https://github.com/freifunkMUC/wg-embed/blob/20165f846d32c03c5517b628890e8eea4bdeca28/pkg/wgembed/iface_userspace.go#L46)

Do we need to adapt those as well?

Reply:
Yes, I noticed them as well and I am not sure. These are the attributes used to create the interface, but are overwritten on interface up. So, it should not really matter.

Hence, I decided against passing the same value into the interface creation functionality and left the default there..

Any opinions?

Reply:
You are right, these are fine for Linux, I did only take a sloppy look... :see_no_evil: 

What about Darwin, though?